sklearn/naive_bayes.py

```diff
@@ -341,8 +374,10 @@ def _partial_fit(self, X, y, classes=None, _refit=False,
             n_classes = len(self.classes_)
             self.theta_ = np.zeros((n_classes, n_features))
             self.sigma_ = np.zeros((n_classes, n_features))
-            self.class_prior_ = np.zeros(n_classes)
-            self.class_count_ = np.zeros(n_classes)
+
+            self.class_count_ = np.zeros(n_classes, dtype=np.float64)
+            self._update_class_prior(class_prior=self.class_prior)
```

Comment:
Why do you need this line? The priors are updated after the mean and variance updates right?